packages/apps/src/app.oauth.ts

```diff
@@ -16,11 +16,11 @@ export async function onTokenExchange<TPlugin extends IPlugin>(
   this: App<TPlugin>,
   ctx: contexts.IActivityContext<ISignInTokenExchangeInvokeActivity, PluginAdditionalContext<TPlugin>>
 ) {
-  const { api, activity, log, next } = ctx;
+  const { api, activity, log, connectionName, next } = ctx;
 
-  if (this.oauth.defaultConnectionName !== activity.value.connectionName) {
+  if (connectionName !== activity.value.connectionName) {
     log.warn(
-      `default connection name "${this.oauth.defaultConnectionName}" does not match activity connection name "${activity.value.connectionName}"`
+      `default connection name "${connectionName}" does not match activity connection name "${activity.value.connectionName}"`
     );
   }
```

Comment:
I think the problem is with binding @rido-min. I think we should still use this.oauth.defaultConnectionName, but i think in `app.ts` this function doesn't bind to `app`, so this.oauth is undefined.